package.json

```diff
@@ -7,18 +7,33 @@
     "test": "tests"
   },
   "dependencies": {
-    "lunr": "^2.0.2",
-    "npm": "^5.0.0"
+    "jquery": "1.12.4",
+    "jquery-ui-dist": "1.12.1",
+    "lunr": "2.3.1",
+    "npm": "^6.0.0"
```

Comment:
Oh yeah.. I forgot you can specify the version of npm in the dependencies like this. I should get in the habit of doing that for everything.